packages/api/src/platforms/vtex/resolvers/searchResult.ts

```diff
@@ -117,16 +117,13 @@ export const StoreSearchResult: Record<string, Resolver<Root>> = {
 
     return filteredFacets
   },
-  metadata: async ({ searchArgs, productSearchPromise }) => {
-    if (!searchArgs.query) {
-      return null
-    }
```

Comment:
question: totally just out of curiosity, why was it like that and why was it removed?

Reply:
We need to handle the category scenario (where there is no query term). If we return null, the fuzzy and operator will be undefined. As far as I know, we should also include these parameters for categories.

Reply:
Got it! Good to know